flood_api/example_code/python/threshold.py

```diff
@@ -0,0 +1,28 @@
+from httpx import Client
+
+with Client() as client:
+    # Get the return period thresholds for the given coordinates
+    response_loc = client.get(
+        url="$api_url$api_path", params={"lon": 22.260536, "lat": 4.882569}
```

Comment:
Simplify all the example-code files by only requiring a single parameter to be replaced. Rename to `endpoint_url`. See comment in `openapi.py`